src/Features/VisualBasic/Portable/Structure/Providers/SelectBlockStructureProvider.vb

```diff
@@ -19,4 +19,46 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Structure
                                type:=BlockTypes.Conditional, isCollapsible:=True))
         End Sub
     End Class
+
+    Friend Class CaseStatementSyntaxStructureProvider
+        Inherits AbstractSyntaxNodeStructureProvider(Of CaseStatementSyntax)
+
+        Protected Overrides Sub CollectBlockSpans(node As CaseStatementSyntax,
+                                                  spans As ArrayBuilder(Of BlockSpan),
+                                                  options As OptionSet,
+                                                  cancellationToken As CancellationToken)
+            Dim _case_ = node.Cases.FirstOrDefault
```

Comment:
this naming is strange.

Reply:
@CyrusNajmabadi 
VB can support multiple case clauses on the same case statement
```VB
Select Case value
  Case 0, 1, 2, 3
```
I think we should only display the first in the banner text. Eg `Case 0, ... `
Renaming to `FirstCaseClauseInCaseStatement`

Reply:
it feels like we should show the entire line, then a ... :)